apps/src/dropletUtils.js

```diff
@@ -158,52 +158,44 @@ standardConfig.categories = {
  * @returns {Array}
```

Comment:
Agree. There are quite a few possible member fields I think, but probably worth while to make an attempt to at least start listing those.